c/alloc.c

```diff
@@ -267,18 +271,24 @@ static int   allocationLengths[ALLOCATIONS_TO_TRACK];
 int showOutstanding(){
   int i;
 
-  printf("total allocations done = %d\n",allocationsTracked);
+  if (MALLOC_TRACE_LEVEL >= 1){
+    printf("total allocations done = %d\n",allocationsTracked);
```

Comment:
Does it mean the loggic needs to be changed? That is, we need to collect the file name and line number of every allocation site we're tracking? If we just add those to this printf call, we'll get something like `alloc.c, 275, showOutstanding`.

Reply:
Well, just for the printf ones. Because we're going to use a formatter for zowelog that does this anyway.

Reply:
As far as I know this function prints information about previously collected allocation sites, if we want to know the line/file/etc of each site, we'll need to change the existing logic in alloc.c. If we just add those macros, you'll always get the same values because this function will get called from the same exact place.

Reply:
Oh I see what you mean.
No, I am not saying we need to make alloc track what function called it.
I was just saying that some of our log messages are bad. The sentence is generic so it makes debugging more annoying.

What I was asking for was:
`(alloc.c:275) total allocations done = 5`
Not:
`(caller.c:1290) total allocations done = 5`
Because this way I know where this short sentence came from. Yes, knowing who calls alloc is nice but that is a different topic.